embedmd/command.go

```diff
@@ -68,20 +68,38 @@ func fields(s string) ([]string, error) {
 	var args []string
 
 	for s = strings.TrimSpace(s); len(s) > 0; s = strings.TrimSpace(s) {
+		var sep int
 		if s[0] == '/' {
-			sep := strings.IndexByte(s[1:], '/')
+			sep = nextSlash(s, 1)
 			if sep < 0 {
 				return nil, errors.New("unbalanced /")
 			}
-			args, s = append(args, s[:sep+2]), s[sep+2:]
```

Comment:
Explanation: We no longer have this special case if we pre-define the offset as 1 in the `nextSlash` call above so can move it outside of the if/else block.